standalone-metastore/pom.xml

```diff
@@ -361,6 +362,12 @@
         <scope>runtime</scope>
         <optional>true</optional>
       </dependency>
+      <!-- check if it introduces any transitive dependencies -->
+      <dependency>
+        <groupId>org.apache.httpcomponents</groupId>
```

Comment:
Did you check if this is bringing any transitive dependencies from the dependency tree?

Reply:
I will check and share an update here.